Contest_API.md

```diff
@@ -610,13 +610,32 @@ The following endpoint is associated with API information:
 
 Properties of version object:
 
+| Name         | Type              | Description
+| :----------- | :---------------- | :----------
+| version      | string            | Version of the API. For this version must be the string `draft`. Will be of the form `<yyyy>-<mm>`, `<yyyy>-<mm>-draft`, or simply `draft`.
+| version\_url | string            | Link to documentation for this version of the API.
+| provider     | provider object ? | Information about the data provider
+
+Properties of the provider object:
+
 | Name         | Type            | Description
 | :----------- | :-------------- | :----------
-| version      | string          | Version of the API. For this version must be the string `draft`. Will be of the form `<yyyy>-<mm>`, `<yyyy>-<mm>-draft`, or simply `draft`.
-| version\_url | string          | Link to documentation for this version of the API.
 | name         | string ?        | Name of this data provider.
+| version      | string ?        | Provider's application version string
+| build\_date  | TIME ?          | TIME provider's application was built
+| type         | string ?        | See Known Provider Types table below
+| doc\_url     | string ?        | Link to provider's documentation
```

Comment:
This is a little ambiguous - is this general documentation for the provider, documentation on how to use the API? How would the client use it?

I would prefer just a general (website) URL for the provider.

Reply:
Some thought about this is required.  Should there be two links (one for API documentation and one for the general website of the client?

Reply:
I'd drop the `doc_url` for now, as long as we don't have agreement on what URL properties we'd like to include and what they should point to.

Reply:
Removed both doc_url and build_date.   build_date can be included in the provider version string if desired.